lib/s3.ts

```diff
@@ -0,0 +1,35 @@
+import { createPresignedPost } from "@aws-sdk/s3-presigned-post";
+import { S3Client, DeleteObjectCommand } from "@aws-sdk/client-s3";
+
+export function createPresigned(uniqueKey: string, contentType: string) {
+  const client = new S3Client({ region: process.env.AWS_REGION });
+
+  return createPresignedPost(client, {
+    Bucket: process.env.AWS_BUCKET_NAME!,
+    Key: uniqueKey,
+    Conditions: [
+      ["content-length-range", 0, 262144000], // up to 250 MB
+      ["starts-with", "$Content-Type", contentType],
+    ],
+    Fields: {
+      acl: "public-read",
+      "Content-Type": contentType,
+    },
+    Expires: 600, // Presigned URL expiration in seconds
+  });
+}
+
+export async function deleteFileFromS3(key: string) {
```

Comment:
can you verify that this is called? I don't see either of the two logs messages below show up

Reply:
I'm seeing it being called: 

![Screenshot 2024-06-26 at 23 36 54](https://github.com/patrickchin/haru2-vercel/assets/65467719/efdbd018-3be9-4156-996c-0bf28da0c8f2)

Are you able to check in the S3 Bucket (I don't have access)?